recipes/pcraster/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set version = "4.3.0" %}
+
+package:
+  name: pcraster
+  version: {{ version }}
+
+source:
+  url: http://pcraster.geo.uu.nl/pcraster/packages/src/pcraster-{{ version }}.tar.bz2
+  sha256: a4be969033d7d08571eca7a5ca27412f3d1a8e1d1ea20d41f6c82e74442c2352
+
+
+build:
+  number: 0
+  skip: True                                # [py2k]
+
+
+  missing_dso_whitelist:
+    - C:/Windows/System32/MSVCP140.dll      # [win]
+    - C:/Windows/System32/VCRUNTIME140.dll  # [win]
+    - C:/Windows/System32/imagehlp.dll      # [win]
+    - C:/Windows/System32/GLU32.dll         # [win]
+    - C:/Windows/System32/OPENGL32.dll      # [win]
+    - C:/Windows/System32/dwmapi.dll        # [win]
+    - C:/Windows/System32/WTSAPI32.dll      # [win]
+
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}         # [linux]
+    - {{ cdt('mesa-dri-drivers') }}         # [linux]
+    - {{ cdt('libselinux') }}               # [linux]
+    - {{ cdt('libxdamage') }}               # [linux]
+    - {{ cdt('libxxf86vm') }}               # [linux]
+    - cmake >=3.15
+    - ninja
+    - make                                  # [win]
+
+  host:
+    - xorg-libxfixes                        # [linux]
+    - boost-cpp
+    - xerces-c
+    - qt
+    - ncurses                               # [not win]
+    - gdal
+    - libgdal
+    - python
+    - numpy 1.14.*
```

Comment:
You don't need the pin here. Please see https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy

Reply:
Sorry, I still do not understand this. I removed the NumPy version entries, and was hoping that 1.14 (i.e. the version in the global pinning) is the version used to build against and thus picked up for the three platforms. For macOS and Windows it seems to work, but on Linux it gives at setup stage:
```
2020-02-25T13:46:14.5891673Z Package numpy conflicts for:
2020-02-25T13:46:14.5892004Z numpy=1.9
2020-02-25T13:46:14.5892473Z gdal=3.0 -> numpy[version='>=1.14.6,<2.0a0']
```
Why numpy=1.9, where is that coming from? Perhaps you can enlighten me in that case... thanks in advance!